stdlib/public/core/ContiguousCollection.swift

```diff
@@ -0,0 +1,74 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// A collection that supports access to its underlying contiguous storage.
+@available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
```

Comment:
```suggestion
@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)
```

`OSX` => `macOS` (11 times) and in same order as rest of stdlib, for easier find and replace.